styles.css

```diff
@@ -90,10 +90,8 @@ footer p {
   margin-top: -10px;
 }
 .large {
```

Comment:
One little change here, which is of course not your fault. I forgot to mention to leave the `margin` property.
So, please add the `margin: 0` to the `.large` class.